### PR TITLE
QAM testsuite - configure branch server

### PR DESCRIPTION
**Pull request by lkotek**
## What does this PR change?

PR add feature and step necessary to configure branch server via `retail_branch_init` command.

## Links

https://github.com/SUSE/spacewalk/issues/11333

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by lkotek** [comment]
I tested this with proxy using real repositories so I can confirm it works. Ports for 4.1 and 4.0 will follow.

File: testsuite/features/qam/init_clients/branch.feature [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# The scenarios in this feature are skipped if there is no proxy
```
Comment (calancha) [inline testsuite/features/qam/init_clients/branch.feature:4]: I see we have the same comment in the feature files at the core.
Probably would be OK to drop those header comments: all the tags do exactly that, neither `@proxy` nor `@private_net` is special.

Comment (lkotek) [reply testsuite/features/qam/init_clients/branch.feature:4]: Good point. I will take a look at this. (Maybe we can change it all at once then.)

**Comment by srbarrios** [comment]
Thanks for that PR!

**Pull request merged**